.github/workflows/linux_meson.yml

```diff
@@ -26,35 +27,66 @@ permissions:
 
 jobs:
   meson_devpy:
-    runs-on: ubuntu-latest
+    name: meson ${{ matrix.os }} / $${ matrix.python_version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest ] # macos-latest ]
+        python_version: [ '3.10' ]
+
+    timeout-minutes: 30
+
     steps:
     - uses: actions/checkout@v3
       with:
         submodules: recursive
         fetch-depth: 0
     - uses: actions/setup-python@v4
       with:
-        python-version: ${{ env.PYTHON_VERSION }}
+        python-version: ${{ matrix.python_version }}
     - name: Install dependencies
       run: |
         pip install -r build_requirements.txt
-        sudo apt-get install -y libopenblas-serial-dev
+    - name: Install pkg-config, OpenBLAS (windows)
+      if: ${{ startswith( matrix.os, 'windows') }}
```

Comment:
Given changes like this and the shell colors line, perhaps a separate CI job file is cleaner?

Reply:
OK, I will set up a separate job